### PR TITLE
inhibition for some cc3test alerts

### PR DESCRIPTION
**Pull request by artherd42**

### CODE REVIEW

File: global/prometheus-alertmanager-operated/templates/am-config-inhibit-rules-cc3test.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+apiVersion: monitoring.coreos.com/v1alpha1
+kind: AlertmanagerConfig
+metadata:
+  name: inhibit-rules-keppel
```
Comment (richardtief) [inline global/prometheus-alertmanager-operated/templates/am-config-inhibit-rules-cc3test.yaml:4]:
```suggestion
  name: inhibit-rules-cc3test
```

**Review by richardtief — APPROVED** [review]
Looks good syntactical. Thanks.

**Review by viennaa — CHANGES_REQUESTED** [review]
Please bump the _minor_ version of the chart since it is adding a new config.

**Review by viennaa — APPROVED** [review]
Good to go

**Pull request merged**